backend/security/src/main/kotlin/io/tolgee/security/authorization/OrganizationAuthorizationInterceptor.kt

```diff
@@ -52,7 +56,16 @@ class OrganizationAuthorizationInterceptor(
     response: HttpServletResponse,
     handler: HandlerMethod,
   ): Boolean {
-    val userId = authenticationFacade.authenticatedUser.id
+    val user = authenticationFacade.authenticatedUserEntity
```

Comment:
Don't use entity here. We don't want to fetch user entity from the DB with every request. Thats why authenticatedUser is DTO, which can be returned from cache.